test/unit/mdl-checkbox/foundation.test.js

```diff
@@ -18,6 +18,9 @@ import test from 'tape';
 import bel from 'bel';
 import lolex from 'lolex';
 import td from 'testdouble';
+
+import {setupFoundationTest} from '../helpers/setup';
```

Comment:
Again, I'm OK with either splitting or not, depending on whether you think it's a problem for our commit message strategy.